lib/stale.js

```diff
@@ -10,28 +10,54 @@ module.exports = class Stale {
 
     await this.ensureStaleLabelExists();
 
-    this.github.paginate(this.getStaleIssues(), data => {
-      data.items.filter(issue => !issue.locked)
-                .forEach(issue => this.mark(issue));
-    });
+    this.mark();
+    this.sweep();
+  }
+
+  mark() {
+    const staleDays = this.config.days || this.config.daysUntilStale;
+
+    var markedIssues = new Set()
+
+    this.getStaleIssuesSearches(staleDays).forEach(search =>
```

Comment:
Previously, this used the blacklisted labels to build a single exclusionary search query, then executed that search to find stale issues. AFAICT there's no inclusionary equivalent to this, so my strategy here is to:

1) get a list of searches that collectively should capture all stale issues (this will be an array of unit length if we're applying an exclusionary strategy, and an array of length `n` if we're applying an inclusionary strategy with `n` labels);
2) perform all those searches;
3) if a resultant issue has not already been marked during this execution, mark it and store it (to avoid marking the same issue as stale multiple times).